spec/hangman_rules_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+require_relative '../lib/hangman_rules'
+
+RSpec.describe HangmanRules do
+
+  let(:words) {Words.new}
+
+  it "returns a random word" do
+    hangman_rules = new_hangman_rules(WordsStub.new)
+
+    word = hangman_rules.random_word()
+
+    expect(word).to eq("hello")
+  end
+
+  it "stores letters correctly guessed" do
```

Comment:
"storing" is such a boring word.
how about "it remembers correctly guessed letters"?